sql/hive/src/test/scala/org/apache/spark/sql/sources/BucketedReadWithHiveSupportSuite.scala

```diff
@@ -18,11 +18,18 @@
 package org.apache.spark.sql.sources
 
 import org.apache.spark.sql.hive.test.TestHiveSingleton
+import org.apache.spark.sql.internal.SQLConf.ADAPTIVE_EXECUTION_ENABLED
 import org.apache.spark.sql.internal.StaticSQLConf.CATALOG_IMPLEMENTATION
 
 class BucketedReadWithHiveSupportSuite extends BucketedReadSuite with TestHiveSingleton {
   protected override def beforeAll(): Unit = {
     super.beforeAll()
+    spark.sessionState.conf.setConf(ADAPTIVE_EXECUTION_ENABLED, false)
```

Comment:
nit: could we use `DisableAdaptiveExecutionSuite` instead?

Reply:
Oh, sure! Thanks.